README.md

```diff
@@ -34,14 +34,28 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. If you were to describe semantic HTML to the next cohort of students, what would you say?
 
+
+-Semantic HTML are HTML tags that clearly give meaning rather than defining it's look.
+
 2. Name two big differences between ```display: block;``` and ```display: inline;```.
 
+-display:block shoves or "bullies" the tags to go under it.
+-display:incline keep it on the same line.
+
+
 3. What are the 4 areas of the box model?
 
+
+-item, padding, margin, border
+
 4. While using flexbox, what axis does the following property work on: ```align-items: center```?
 
+-main axis
+
 5. Explain why git is valuable to a team of developers.
 
+-Because it's beneficial to dev's to go through different repo's and select. It's also beneficial to house content on github which helps to save and commit.
```

Comment:
Great answer, Git is a powerful tool that has become industry standard, i know it may feel confusing now but it is a crucial skill that you will develop further as you keep using it more and more.